portafly/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:10-alpine as builder
+WORKDIR /tmp/portafly
+COPY . ./
+ENV PUBLIC_URL="/portafly"
+RUN yarn install \
```

This will install only production dependencies, according to our yarn.lock file
```suggestion
RUN yarn install --production --frozen-lockfile \
```